pixl_dcmd/src/pixl_dcmd/main.py

```diff
@@ -60,20 +61,26 @@ def anonymise_dicom(dataset: Dataset) -> Dataset:
     - applying tag operations based on the config file
     Returns anonymised dataset.
     """
-    slug = dataset.get_private_item(
+    raw_slug = dataset.get_private_item(
         DICOM_TAG_PROJECT_NAME.group_id,
         DICOM_TAG_PROJECT_NAME.offset_id,
         DICOM_TAG_PROJECT_NAME.creator_string,
     ).value
-    project_config = load_project_config(slug)
-    logger.error(f"Received instance for project {slug}")
+    # Get both strings and bytes, which is fun
+    if isinstance(raw_slug, bytes):
+        logger.debug(f"Bytes slug {raw_slug!r}")
+        slug = raw_slug.decode("utf-8").strip()
+    else:
+        logger.debug(f"String slug '{raw_slug}'")
+        slug = raw_slug
```

Comment:
Is this a DICOM thing? 🫠

Reply:
Its certainly not documented, not quite sure why it happens only sometimes